docs/course-info/setup.rst

```diff
@@ -273,3 +273,42 @@ while being in the root directory of the repository and click on the appearing l
   therefore SuperCollider is omitted in the docker image.
 
   Any help on this is appreciated.
+
+
+Updating and working with the repository
+----------------------------------------
+
+The course material is stored and distributed via `git <https://git-scm.com/>`__.
+As there are constantly updates on the course material there will be conflicts
+when we have worked locally on a notebook as *git* does not know how to resolve
```

Comment:
easier to parse:
… when we have worked locally on a notebook. This is because *git* …